conda/cli/main_search.py

```diff
@@ -107,41 +108,25 @@ def configure_parser(sub_parsers):
 
 
 def execute(args, parser):
-    from ..base.context import context
-    from ..core.index import get_channel_priority_map
-    from ..exceptions import PackagesNotFoundError, ResolvePackageNotFound
-
     try:
         execute_search(args, parser)
-    except ResolvePackageNotFound as e:
```

Comment:
Since the new MatchSpec in 4.4., `e.bad_deps` formatting doesn't work the same in search as it does in install.